src/components/scenes/LandingScene.tsx

```diff
@@ -20,8 +20,12 @@ interface Props extends SceneProps<'landing'> {
 
 export const LandingScene = (props: Props) => {
   const dispatch = useDispatch()
-  const { initialUserInfo } = useImports()
-  const { onLogEvent } = useImports()
+  const { initialUserInfo, experimentConfig, onLogEvent } = useImports()
+
+  const createAccTextVar =
```

Comment:
This and line 26-28 should be put in a common routine in a file like experiments.ts